change/@internal-react-composites-950d55c1-62f5-4248-a46d-bbde257e51ad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Local and remote PIP not showing local video tile if no permissions to camera",
```

Comment:
instead of "no permissions" to camera can we use "if the role does not permit camera" - reason being is I read that as if the browser has blocked permissions to the camera